src/doc/reference.md

```diff
@@ -2994,7 +2994,7 @@ Some examples of call expressions:
 # fn add(x: i32, y: i32) -> i32 { 0 }
 
 let x: i32 = add(1i32, 2i32);
-let pi: Option<f32> = "3.14".parse().ok();
+let pi: Option<f32> = "3.14".parse();
```

Comment:
When was it updated? [The documentation](http://doc.rust-lang.org/std/str/trait.StrExt.html#tymethod.parse) still seems to refer to it returning an Option<F>.